iDiff/tests/multi_hist_diff_expected.json

```diff
@@ -5,7 +5,7 @@
       "Image1": "gcr.io/gcp-runtimes/pip-base",
       "Image2": "gcr.io/gcp-runtimes/pip-modified",
       "Adds": [
-        "/bin/bash"
+        "/bin/bash\n"
```

Comment:
this is json output, right? can we strip the newlines before adding it to the output struct?